music21/metadata/__init__.py

```diff
@@ -2358,6 +2307,9 @@ def _convertValue(namespaceName: str, value: t.Any) -> ValueType:
             raise exceptions21.MetadataException(
                 f'invalid type for Contributor: {type(value).__name__}')
 
+        if valueType is int:
+            return int(value)
```

Comment:
aha, this seems like the one place where the return type `t.Union[ValueType, int]` seems better to use rather than including `int` in the definition of ValueType.